### PR TITLE
Updated TF 6534

### PR DESCRIPTION
**Pull request by SeanGallardy**
Changed verbiage to denote that it's applicable in all versions since 2012 up to and including 2019.

### CODE REVIEW

**Comment by PRMerger15** [comment]
@SeanGallardy : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: docs/t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ The following table lists and describes the trace flags that are available in [!
...
-|<a name="tf6534"></a>**6534**|Enables performance improvement of query operations with spatial data types in [!INCLUDE[ssSQL11](../../includes/sssql11-md.md)], [!INCLUDE[ssSQL14](../../includes/sssql14-md.md)] and [!INCLUDE[sssql16-md](../../includes/sssql16-md.md)]. The performance gain will vary, depending on the configuration, the types of queries, and the objects. For more information, see this [Microsoft Support article](https://support.microsoft.com/kb/3107399).<br /><br />**Scope**: global only|   
+|<a name="tf6534"></a>**6534**|Enables performance improvement of query operations with spatial data types in [!INCLUDE[ssSQL11](../../includes/sssql11-md.md)] through [!INCLUDE[sssql16-md](../../includes/sssql19-md.md)]. The performance gain will vary, depending on the configuration, the types of queries, and the objects. For more information, see this [Microsoft Support article](https://support.microsoft.com/kb/3107399).<br /><br />**Scope**: global only|
```
Comment (MikeRayMSFT) [inline docs/t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql.md:146]:
```suggestion
|<a name="tf6534"></a>**6534**|Enables performance improvement of query operations with spatial data types beggining with [!INCLUDE[ssSQL11](../../includes/sssql11-md.md)]. The performance gain will vary, depending on the configuration, the types of queries, and the objects. For more information, see this [Microsoft Support article](https://support.microsoft.com/kb/3107399).<br /><br />**Scope**: global only|   
```

**Comment by PRMerger-2** [comment]
@MikeRayMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ The following table lists and describes the trace flags that are available in [!
...
-|<a name="tf6534"></a>**6534**|Enables performance improvement of query operations with spatial data types in [!INCLUDE[ssSQL11](../../includes/sssql11-md.md)], [!INCLUDE[ssSQL14](../../includes/sssql14-md.md)] and [!INCLUDE[sssql16-md](../../includes/sssql16-md.md)]. The performance gain will vary, depending on the configuration, the types of queries, and the objects. For more information, see this [Microsoft Support article](https://support.microsoft.com/kb/3107399).<br /><br />**Scope**: global only|   
+|<a name="tf6534"></a>**6534**|Enables performance improvement of query operations with spatial data types beggining with [!INCLUDE[ssSQL11](../../includes/sssql11-md.md)]. The performance gain will vary, depending on the configuration, the types of queries, and the objects. For more information, see this [Microsoft Support article](https://support.microsoft.com/kb/3107399).<br /><br />**Scope**: global only|
```
Comment (MikeRayMSFT) [inline docs/t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql.md:146]: [](url)
```suggestion
|<a name="tf6534"></a>**6534**|Enables performance improvement of query operations with spatial data types beginning with [!INCLUDE[ssSQL11](../../includes/sssql11-md.md)]. The performance gain will vary, depending on the configuration, the types of queries, and the objects. For more information, see this [Microsoft Support article](https://support.microsoft.com/kb/3107399).<br /><br />**Scope**: global only|   
```

**Comment by PRMerger19** [comment]
@MikeRayMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by MikeRayMSFT** [comment]
Updated to note that it begins with 2012 (implying that support will continue through vNext). 

#sign-off

**Comment by PRMerger18** [comment]
@MikeRayMSFT: I'm sorry - only the author of this article, @pmasl, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Review by MikeRayMSFT — APPROVED** [review]

**Pull request merged**